iOS Example Frame/iOS Example Frame/ViewController/SecurityCodeViewController.swift

```diff
@@ -0,0 +1,88 @@
+// 
+//  SecurityCodeViewController.swift
+//  iOS Example Frame
+//
+//  Created by Okhan Okbay on 28/09/2023.
+//  Copyright © 2023 Checkout. All rights reserved.
+//
+
+import Frames
+import UIKit
+
+final class SecurityCodeViewController: UIViewController {
+
+  @IBOutlet private weak var defaultSecurityCodeComponent: SecurityCodeComponent!
+  @IBOutlet private weak var defaultPayButton: UIButton!
+
+  @IBOutlet private weak var customSecurityCodeComponent: SecurityCodeComponent!
+  @IBOutlet private weak var customPayButton: UIButton!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    setupNavigationBar()
+  }
+
+  override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+
+    navigationController?.setNavigationBarHidden(false, animated: animated)
+
+    setupDefaultSecurityCodeComponent()
+    setupCustomSecurityCodeComponent()
+  }
+
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+
+    defaultSecurityCodeComponent.becomeFirstResponder()
+  }
+
+  func setupDefaultSecurityCodeComponent() {
+    let configuration = SecurityCodeComponentConfiguration(serviceAPIKey: Factory.apiKey,
+                                                  environment: Factory.environment,
+                                                  cardScheme: .mastercard)
```

Comment:
From merchant integration, they will have any scheme as a string value. So, in Android, if merchant would like to add cardScheme enum, we provide the function to the merchant name as `CardScheme.fromString(cardSchemeValue = schemeStringValue)` to map the scheme string value to the enum. Curious, in ios do we need this mapping function for merchant to load SDK Cardscheme enum from string value?

Reply:
Looks like it was added before. I think they will need to lowercase the scheme strings on iOS before passing them i in @chintan-soni-cko 

<img width="520" alt="Screenshot 2023-10-11 at 11 55 41" src="https://github.com/checkout/frames-ios/assets/125963311/54eb7bad-c7b1-4ace-8b72-fee39fb555ca">

Reply:
In Android, we managed through the function `CardScheme.fromString`. So, if they add uppercase, lower case or with underscore. We return proper value and in this case merchants don't need to write specific uppercase or lowercase string value to map the enum from SDK.

Reply:
ah got it! same here. they can pass lowercase or uppercase

Reply:
Additionally, we managed for underscore as well. Refer to unit tests for more visibility [here](https://github.com/checkout/frames-android/blob/master/checkout/src/test/java/com/checkout/base/model/CardSchemeTest.kt)

Reply:
Do they get underscored string representations of card schemes from our API? @chintan-soni-cko

Reply:
what was your initial motivation to handle underscored string representations of card scheme names in this enum? Did someone mention it or did you see it anywhere? or just as a precaution? @chintan-soni-cko

Reply:
Just a precaution covering maximum cases from the SDK side. However, I have just seen it. Last time they called american_express as "AMERICAN EXPRESS". Currently, in Sandbox they are returning "AMEX". So, it won't work with SDK for the card Scheme. What's your thought?
![Screenshot 2023-10-17 at 17 34 04](https://github.com/checkout/frames-ios/assets/114917119/0964d065-8ef7-4f5b-88a3-c6f17617cd29)

Reply:
I would hesitate to handle the `underscored string` case if we can't see it happening @chintan-soni-cko

Reply:
`amex`, `AMEX`, `American Express` and lowercase+uppercase variations are being handled atm on iOS @chintan-soni-cko

Reply:
I think this is not needed anymore. resolving this @chintan-soni-cko feel free to re-open please